gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPicker.kt

```diff
@@ -90,6 +89,10 @@ internal fun AvatarPicker(
                                 duration = SnackbarDuration.Long,
                             )
                         }
+
+                        is AvatarPickerAction.LaunchImageCropper -> {
+                            uCropLauncher.launchAvatarCrop(action.imageUri, action.tempFile, context)
```

Comment:
Should we introduce some abstraction layer here in order to reduce coupling with a third party library?

Reply:
Done! Let me know if this is what you meant!